.github/ISSUE_TEMPLATE/issue_template.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Issue report
+about: Report an issue in the WordPress documentation and help us improve the documentation.
+labels: tracking issue, needs triage
+---
+
+---
+<!--
+Please fill out ALL required sections. Issue reports with missing information will
+be closed.
```

Comment:
Not all sections are required (the last one isn't for example, they may not have a suggested fix)

So: 

if all sections are required, this should be "Please fill out all sections"

If some sections are optional/not required, then the ones that *are* required should have that indicated

Reply:
"Issue reports with missing information will be closed."

This sounds like lingo to me. How about

We can't work on fixing a problem without all the required sections filled in. So please be sure your submission is complete; if it's not, it will be marked as incomplete, and closed without being fixed.